bcipy/display/components/task_bar.py

```diff
@@ -0,0 +1,176 @@
+"""Task bar component"""
+
+from typing import Dict, List
+from psychopy import visual
+from psychopy.visual.basevisual import BaseVisualStim
+from psychopy.visual.line import Line
+import bcipy.display.components.layout as layout
+
+
+class TaskBar:
+    """Component for displaying task-related information in a task window. The
+    component elements are positioned at the top of the window.
+
+    Parameters
+    ----------
+        win - visual.Window on which to render elements
+        colors - Ordered list of colors to apply to task stimuli
+        font - Font to apply to all task stimuli
+        height - Height of all task text stimuli
+        text - Task text to apply to stimuli
+    """
+
+    def __init__(self,
+                 win: visual.Window,
+                 colors: List[str] = None,
+                 font: str = 'Courier New',
+                 height: float = 0.1,
+                 text: str = ''):
+        self.colors = colors or ['white']
+        self.font = font
+        self.height = height
+        self.text = text
+        self.layout = layout.at_top(layout.WindowContainer(win),
+                                    self.compute_height())
+        self.stim = self.init_stim()
+
+    def compute_height(self):
+        """Computes the component height using the provided config."""
+        return self.height + 0.05
```

Comment:
Why 0.05?

Reply:
Would a limit here make sense? Or maybe a warning is given about height to window size? I know we allow any value, which would impact more than just this component! We can talk about user inputs again later if you'd like. I keep returning to handling it more internally and offering discrete options (S, M, L text size; dark, light, colorblind mode).

Reply:
The 0.05 value was determined by trial-and-error for padding. I'll add some documentation here.

I think making the height a more discrete value is a good idea. It's hard to think in terms of 'norm' units and it would be a better way for the users changing these parameters to visualize what they mean. Let's revisit this in a followup ticket.